Source/Core/Core/ConfigManager.cpp

```diff
@@ -249,6 +249,8 @@ void SConfig::SaveCoreSettings(IniFile& ini)
   core->Set("OverrideGCLang", bOverrideGCLanguage);
   core->Set("DPL2Decoder", bDPL2Decoder);
   core->Set("Latency", iLatency);
+  core->Set("AudioStretch", m_audio_stretch);
```

Comment:
It's a little bit useful, but on the other hand, it would be nice to include "Audio" in the name since there's no other way to tell that this is audio related (since it's in the Core section). And besides, we sometimes break compatibility with our own INI names...